### PR TITLE
fixing menu background color

### PR DESCRIPTION
**Pull request by RachellCalhoun**
fix menu background color and font color

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
Hey @jinpark when you get  a chance can you look at this?

File: content/static/css/content.css [hunk 1]
```diff
@@ -37,17 +39,22 @@ body {
   color: #fff;
   text-decoration: underline;
 }
+.navbar-default .navbar-nav .open .dropdown-menu > li > a:focus, .navbar-default .navbar-nav .open .dropdown-menu > li > a:hover {
+  background-image: none;
```
Comment (jinpark) [inline content/static/css/content.css]: i assume it doesnt work without this set as `none`?

File: content/static/css/content.css [hunk 2]
```diff
@@ -37,17 +39,22 @@ body {
...
 .navbar-default .navbar-nav > .active > a,.navbar-default .navbar-nav > .active > a:hover {
   background-image: none;
   background-color: #7a25ae;
   color: #fff;
 }
-
-.navbar-nav > li {
-  padding-left: 1em;
-  padding-right: 1em;
+.navbar .navbar-nav li a:focus,.navbar .navbar-nav > li > a:hover {
```
Comment (jinpark) [inline content/static/css/content.css]: space after comma

File: content/static/css/content.css [hunk 3]
```diff
@@ -59,11 +66,26 @@ body {
...
 
-.navbar-default .navbar-nav .open .dropdown-menu > li > a:focus,.navbar-default .navbar-nav .open .dropdown-menu > li > a {
+.navbar-default .navbar-nav .open .dropdown-menu > li > a {
   color: #fff;
 }
 
 /*general*/
+.blue-background {
+  background: rgba(116, 72, 236, 0.67);
+  /*padding-right:20px;
```
Comment (jinpark) [inline content/static/css/content.css]: remove comments

**Comment by RachellCalhoun** [comment]
:+1: I fixed up all you mentioned and Found another mistake where focus of li was black. I just removed it.

**Comment by jinpark** [comment]
lgtm

**Pull request merged**